src/components/customerCases/customerCase/CustomerCase.tsx

```diff
@@ -44,7 +44,10 @@ export default async function CustomerCase({
           />
         </div>
         <div className={styles.mainImageWrapper}>
-          <SanitySharedImage image={customerCase.image} />
+          <SanitySharedImage
+            image={customerCase.image}
+            showFigureDescription={true}
```

Comment:
Hva er det som setter showFigureDescription til true? Er det hvis det er med bildebeskrivelse?

Reply:
Jeg lurer på om vi kanskje burde fjerne dette og heller la det være opp til brukeren å legge det til. Akkurat nå er det  mulig å legge til bildetekst overalt hvor bilder legges inn, og jeg er usikker på om vi ønsker å ha denne muligheten for alle bilder. For eksempel i inngangsmodulen til customer case, så vil vi vel strengt tatt ikke ha bildetekst. Men det er også rart for brukeren om man legger ved bildetekst, også forstår man ikke hvor bildeteksten blir av. Det var grunnen til at jeg nevnte i kommentaren på PR-en at denne løsningen kanskje ikke er den beste.

Reply:
Ja for jeg tror nok at alle bildene på customer case trenger kreditering, men som du sier ikke på inngangsmodulen